src/api/api.cc

```diff
@@ -10909,6 +10912,28 @@ void RecordReplayOnExceptionUnwind(Isolate* isolate) {
   if (!isolate->is_catchable_by_javascript(*exception))
     return;
 
+  {
+    // Note: Most of this is copied from |ComputeLocation| in messages.cc.
+    JavaScriptFrameIterator it(isolate);
+    if (!it.done()) {
+      // Compute the location from the function and the relocation info of the
+      // baseline code. For optimized code this will use the deoptimization
+      // information to get canonical location information.
+      std::vector<FrameSummary> frames;
+      it.frame()->Summarize(&frames);
+      auto& summary = frames.back().AsJavaScript();
```

Comment:
Paranoid, but why not just check if frames > 0 here.

Reply:
This is copied from `ComputeLocation` in `messages.cc`. I don't feel too worried about that.

Reply:
I'm guessing its implied by `if (!it.done())` 🤔